### ISSUE TITLE
Bungee-based Plugin Locales don't translate properly on an offline mode network

### ISSUE THREAD

**Issue by SheepBroadcast** [tritonmc/Triton#277]
### Describe the bug

I'm running a Bungee Network with `waterfall-1.19-504` and five Paper servers running `Paper version git-Paper-139 (MC: 1.19.2)`.
My network is set to `online-mode: false`. Triton is set to MySQL storage, which is on the Bungee server.
So far we've translated almost everything on the Paper servers using Triton and TWIN and it works just fine when switching between English and the second language we have.
However, we got to the point where we want to translate a few Bungee-based plugins that we have (two of them being BungeeAnnounce and SkinsRestorer) and for some reason sometimes their locales get defaulted to English in-game, regardless of how a player has set their language preference. Example: Two people have their language set to non-English - one sees BungeeAnnounce's broadcasts in non-English, the other one sees them in English.

### How can the bug be reproduced?

Translate a Bungee-based plugin in another language on a `online-mode:false` Bungee network and try out the translations.

### Expected behaviour

It was expected for the Bungee-based plugin locales to work like the Spigot-based plugin locales.

### Which Spigot fork are you using?

PaperMC

### What's your server version?

Paper version git-Paper-139 (MC: 1.19.2)

### Which proxy are you using?

Waterfall

### What's your proxy version?

waterfall-1.19-504

### What's the MC version of your client?

1.19.2

### What's the Triton version you're using?

3.8.2

**Comment by diogotcorreia** [comment]
Hi!

Thank you for the detailed description, it helps a lot!
I've tried the exact setup you've described, but unfortunately it works correctly for me.

Have you tried setting the log level to 2 (through the config or using the `triton loglevel 2` command)? It prints out some extra information that might help me figure out where the problem is.

Here is an example of information that would be useful to me:
![image](https://user-images.githubusercontent.com/7467891/218815326-712c11f9-96b6-40c0-b6b0-b209714d9700.png)

```
[17:37:52 INFO]: [/127.0.0.1:49636] <-> InitialHandler has connected
[17:37:52 INFO] [Triton]: [TRACE] [MySQL Storage] Getting language for player BungeeLanguagePlayer{uuid=a3fb6eab-fe26-3682-bedb-44871c17ef6b, language=null}
[17:37:52 INFO] [Triton]: [DEBUG] Saving language for 127.0.0.1...
[17:37:52 INFO] [Triton]: [DEBUG] Saved language for 127.0.0.1!
[17:37:52 INFO] [Triton]: [TRACE] [Player Manager] Registering language player BungeeLanguagePlayer{uuid=a3fb6eab-fe26-3682-bedb-44871c17ef6b, language=en_GB}
[17:37:52 INFO] [Triton]: [TRACE] Injecting pipeline for player BungeeLanguagePlayer{uuid=a3fb6eab-fe26-3682-bedb-44871c17ef6b, language=en_GB}
[17:37:53 INFO]: [Rexcantor64|/127.0.0.1:49636] <-> ServerConnector [lobby] has connected
[17:37:53 INFO] [Triton]: [TRACE] Player BungeeLanguagePlayer{uuid=a3fb6eab-fe26-3682-bedb-44871c17ef6b, language=en_GB} connected to a new server
[17:37:53 INFO] [Triton]: [TRACE] Sending player BungeeLanguagePlayer{uuid=a3fb6eab-fe26-3682-bedb-44871c17ef6b, language=en_GB} language to server lobby
[17:37:53 INFO] [Triton]: [TRACE] Sending player BungeeLanguagePlayer{uuid=a3fb6eab-fe26-3682-bedb-44871c17ef6b, language=en_GB} language to server lobby
[17:37:53 INFO] [Triton]: [DEBUG] Saving language for a3fb6eab-fe26-3682-bedb-44871c17ef6b...
[17:37:53 INFO] [Triton]: [TRACE] Executing language commands for player BungeeLanguagePlayer{uuid=a3fb6eab-fe26-3682-bedb-44871c17ef6b, language=en_GB}
[17:37:53 INFO] [Triton]: [DEBUG] Saved language for a3fb6eab-fe26-3682-bedb-44871c17ef6b!
[17:38:05 INFO]: Rexcantor64 executed command: /announce [lang]chat.test.0[/lang]
[17:38:05 INFO] [Triton]: [TRACE] Trying to get translation with key 'chat.test.0' in language 'en_GB'
[17:38:05 INFO] [Triton]: [TRACE] Found translation with key 'chat.test.0' in language 'en_GB'
[17:38:08 INFO]: Rexcantor64 executed command: /lang
[17:38:08 INFO] [Triton]: [TRACE] Forwarding command '' from player 'a3fb6eab-fe26-3682-bedb-44871c17ef6b'
[17:38:10 INFO] [Triton]: [TRACE] Executing language commands for player BungeeLanguagePlayer{uuid=a3fb6eab-fe26-3682-bedb-44871c17ef6b, language=pt_PT}
[17:38:10 INFO] [Triton]: [DEBUG] Saving language for a3fb6eab-fe26-3682-bedb-44871c17ef6b...
[17:38:10 INFO] [Triton]: [DEBUG] Saved language for a3fb6eab-fe26-3682-bedb-44871c17ef6b!
[17:38:15 INFO]: Rexcantor64 executed command: /announce [lang]chat.test.0[/lang]
[17:38:15 INFO] [Triton]: [TRACE] Trying to get translation with key 'chat.test.0' in language 'pt_PT'
[17:38:15 INFO] [Triton]: [TRACE] Found translation with key 'chat.test.0' in language 'pt_PT'
```

**Comment by diogotcorreia** [comment]
Closing this as stale. Will reopen if someone finds a way to reproduce it. It's also possible that this was fixed by #388.
